.github/workflows/ci.yml

```diff
@@ -11,7 +11,6 @@ jobs:
       matrix:
         npm-test:
         - i18n_extract
-        - is-es5
         - lint
         - test
```

Comment:
Oooh. You know, maybe this isn't the right forum... but should we be requiring any of these test suites to pass before a PR can be merged? It doesn't look like we're requiring any of these today, but should be easy to add within GitHub (in the Branch Protection rules)

Reply:
(asking because I thougth that we might need to remove the `is-es5` tests as required in the Branch Protection settings)

Reply:
That is a thinker... maybe a quick parking lot?